3p/iframe-messaging-client.js

```diff
@@ -40,15 +37,29 @@ export class IframeMessagingClient {
   }
 
   /**
+   * Make an event listening request to the host window.
+   *
+   * @param {string} requestType The type of the request message.
+   * @param {string} responseType The type of the response message.
+   * @param {function(object)} callback The callback function to call
+   *   when a message with type responseType is received.
+   */
+  makeRequest(requestType, responseType, callback) {
+    const unlisten = this.registerCallback(responseType, callback);
```

Comment:
the use case here (observe intersection) actually expects multiple invocation of the callback.